Dockerfile.QA

```diff
@@ -61,10 +63,10 @@ RUN cmake -DTRITON_ENABLE_GPU=ON \
 # Add inception_graphdef model to example repo
 WORKDIR /workspace/docs/examples/model_repository
 RUN mkdir -p inception_graphdef/1 && \
-    wget -O /tmp/inception_v3_2016_08_28_frozen.pb.tar.gz \
+    wget -O ${TRITONBUILD_DIR}/../inception_v3_2016_08_28_frozen.pb.tar.gz \
         https://storage.googleapis.com/download.tensorflow.org/models/inception_v3_2016_08_28_frozen.pb.tar.gz && \
-    (cd /tmp && tar xzf inception_v3_2016_08_28_frozen.pb.tar.gz) && \
-    mv /tmp/inception_v3_2016_08_28_frozen.pb inception_graphdef/1/model.graphdef
+    (cd ${TRITONBUILD_DIR}/.. && tar xzf inception_v3_2016_08_28_frozen.pb.tar.gz) && \
+    mv ${TRITONBUILD_DIR}/../inception_v3_2016_08_28_frozen.pb inception_graphdef/1/model.graphdef
```

Comment:
This assumes `TRITONBUILD_DIR` has a parent folder, should just create temporary location within and clean up afterwards

Reply:
Removed this assumption by using /tmp as default to make it consistent with tmp dir usage in build.py